core/src/cluster_nodes.rs

```diff
@@ -237,14 +239,24 @@ impl ClusterNodes<RetransmitStage> {
             .position(|node| node.pubkey() == self.pubkey)
             .unwrap();
         if drop_redundant_turbine_path(shred.slot(), root_bank) {
+            let root_distance = match self_index {
+                0 => 0,
+                1..=200 => 1,
+                _ => 2,
+            };
             let peers = get_retransmit_peers(fanout, self_index, &nodes);
-            return (Vec::default(), peers.collect());
+            return (root_distance, Vec::default(), peers.collect());
         }
+        let root_distance = match self_index {
+            0 => 0,
+            1..=199 => 1,
```

Comment:
I think we should be using the `DATA_PLANE_FANOUT - 1` constant here

Reply:
done